File: cms-sm/src/main/java/com/sophia/cms/sm/model/SchemaTableModel.java
package com.sophia.cms.sm.model;

import lombok.Data;

import java.io.Serializable;
import java.util.List;

/**
 * Created by lenovo on 2018/2/12.
 */
@Data
public class SchemaTableModel implements Serializable{

    private String tableName;
    private String tableComment;
    private List<SchemaColumnModel> schemaColumnModelList;
}

File: client/src/app/sm/model/schematable.model.ts
import { FieldModel } from './field.model';
import { SchemaColumnModel } from './schemacolumn.model';

export class SchemaTableModel {
  constructor() {
    this.tableName = '';
    this.tableComment = '';
    this.schemaColumnModelList = new Array<SchemaColumnModel>();
  }
  tableName: string;
  tableComment: string;
  schemaColumnModelList: Array<SchemaColumnModel>;
}
